WcaOnRails/app/webpacker/stylesheets/homepage.scss

```diff
@@ -0,0 +1,82 @@
+// Override the navbar border bottom just for the homepage.
+.navbar.navbar-static-top {
+  border: 0;
+}
+
+// Override the body background color so that the homepage content looks alright.
+body {
+  background-color: #f8f8f8;
+}
+
+// Override the footer alignment as we don't have the same container width as
+// on other pages.
+footer {
+  .container.footer-links {
+    text-align: center;
+  }
+}
+
+#homepage {
+  // NOTE:
+  //   - We set top/bottom negative margins because on the rest of the website
+  //   we have a margin between the navigation and the actual content.
+  //   - We set right/left margins to auto to center the content on wide screens.
+  margin-top: -15px;
+  margin-bottom: -10px;
+  #cover-homepage {
+    background: url('../images/homepage_banner.svg') no-repeat center center fixed;
+    background-color: #f8f8f8;
+    -webkit-background-size: cover;
+    -moz-background-size: cover;
+    -o-background-size: cover;
+    background-size: cover;
+
+    min-height: 500px;
+    width: 100%;
+    padding: 1em 0em;
+    .ui.header {
+      font-family: Roboto, 'Helvetica Neue',Arial,Helvetica,sans-serif;
```

Comment:
Do we want to import this font or does it rather try to target on of fonts the user has installed?

Reply:
This `font-family` is actually not needed anymore since Roboto is the default font for the website, but I believe your comment still apply generally!

I actually don't know how the browser makes its selection: as far as we're concerned the font face is defined here in our FUI fork: https://github.com/thewca/Fomantic-UI/blob/wca/src/themes/wca/globals/site.overrides.
Do you know if it tries to load Roboto first from the system font or if it uses our font face no matter what?
(and follow-up: do you know if (and how) it is possible to make it look for the font in system fonts and only then make it use our font face ?)

Reply:
I was just wondering if we actually load the Roboto font somewhere, so the fork link definitely clarifies that!

As for the questions, I believe [this documentation page](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face#Description) provides the answer. As you can see we use `local` as the primary font source [here](https://github.com/thewca/Fomantic-UI/blob/0feaaa3a6056ab21409a2551826cb4186ff4f9c6/src/themes/wca/globals/site.overrides#L6-L11) and only fallback to fetching the font.

Reply:
Awesome thanks for the details!